docs/hubs-features.md

```diff
@@ -119,14 +116,14 @@ A menu of emoji spawners is displayed when you display [object or avatar menus](
 The emoji menu lets you spawn a grabbable emoji that emits particles when shaken. The emoji is deleted a few seconds after you let go of it.
 
 
-## Share Room Dialog
+## Invite Menu
 
-The "Share" button opens a dialog with the information you need in order to share the room with others, so that they can join you in the room.
+The "Invite" button opens a dialog with the information you need in order to share the room with others, so that they can join you in the room.
 
-![Enter room on Wired in device](img/hubs-sharing-dialogue.jpeg)
+![Enter room on Wired in device](img/hubs-invite-dialogue.png)
 
 You can copy or share the hub.link URL or use the numeric code to share a link verbally. Note that numeric codes expire after 72 hours.
 
 The `iframe` HTML code (displayed if you have room permissions) lets you embed the room in a web page.
 
-The "notify me" checkbox (displayed if you have room permissions) enables you to sign up for notifications that alert you when another user enters the room. You can sign up for notifications on your phone or desktop. You do not need to remain in the room to receive notifications.
+<!-- The "notify me" checkbox (displayed if you have room permissions) enables you to sign up for notifications that alert you when another user enters the room. You can sign up for notifications on your phone or desktop. You do not need to remain in the room to receive notifications. -->
```

Comment:
Do you want to delete these comments since they're not needed anymore?

Reply:
I'm going to keep them in as comments for the time being in the event we re-add the feature back in